api/src/main/java/uk/gov/dhsc/htbhf/claimant/entity/PaymentCycle.java

```diff
@@ -0,0 +1,87 @@
+package uk.gov.dhsc.htbhf.claimant.entity;
+
+import lombok.*;
+import uk.gov.dhsc.htbhf.claimant.entitlement.PaymentCycleVoucherEntitlement;
+import uk.gov.dhsc.htbhf.claimant.entity.converter.ListOfLocalDatesConverter;
+import uk.gov.dhsc.htbhf.claimant.entity.converter.PaymentCycleVoucherEntitlementConverter;
+import uk.gov.dhsc.htbhf.eligibility.model.EligibilityStatus;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+import static java.util.Collections.unmodifiableSet;
+
+@Entity
+@Table(name = "payment_cycle")
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+@EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = true)
+@ToString(callSuper = true)
+public class PaymentCycle extends BaseEntity {
+
+    @Version
+    @Column(name = "version_number")
+    private Integer versionNumber;
+
+    @NotNull
+    @ManyToOne(fetch = FetchType.LAZY)
+    @ToString.Exclude
+    private Claim claim;
```

Comment:
Why do we have the `claimId` on the `Payment` but the `Claim` on the `PaymentCycle`, any particular reason they are different?

Reply:
That was a mistake, changes this to be a Claim.